cpu/esp_common/freertos/Kconfig

```diff
@@ -10,6 +10,8 @@ if TEST_KCONFIG && HAS_ARCH_ESP
 config MODULE_ESP_FREERTOS
     bool
     default y
+    select MODULE_ZTIMER
+    select MODULE_ZTIMER_MSEC
```

Comment:
Please remove them, shouldn't be needed.

Reply:
I'm confused, doesn't `task` use ztimer?

Reply:
`ztimer` is only used in function `vTaskDelay` which in turn is only used by the following files in vendor code and binary libraries for network interfaces:

`cpu/esp32/vendor/esp-idf/esp32/wifi_os_adapter.c`
`cpu/esp32/vendor/esp-idf/ethernet/emac_main.c`
`cpu/esp8266/vendor/esp-idf/esp8266/source/esp_wifi_os_adapter.c`

That is, it is sufficient that `vTaskDelay` is compilable, but it is not linked. So the `ztimer` module isn't needed as long as network interface aren't used.

Reply:
I had missed the ifdef, thanks!